mycli/main.py

```diff
@@ -1120,6 +1151,33 @@ def cli(database, user, host, port, socket, password, dbname,
         if not port:
             port = uri.port
 
+    if ssh_config_host:
+        if not paramiko:
+            click.secho(
+                "Cannot use SSH transport because paramiko isn't installed, "
+                "please install paramiko or don't use --ssh-config_host=",
+                err=True, fg='red'
+            )
+            exit(1)
+        try:
+            ssh_config = paramiko.config.SSHConfig().from_path(ssh_config_path)
+        except paramiko.ssh_exception.ConfigParseError as err:
+            click.secho('Invalid SSH configuration file. '
+                        'Please check the SSH configuration file.',
+                        err=True, fg='red')
+            exit(1)
+        except FileNotFoundError as e:
```

Comment:
This error will also be raised if the user doesn't specify `ssh_config_path` and doesn't have the default `$HOME/.ssh/config`

Reply:
I think that's intended because the default config path is `$HOME/.ssh/config` and user shouldn't `ssh_config_host` unless there is a valid path to the config file.